src/test/java/org/folio/des/service/JobServiceTest.java

```diff
@@ -53,6 +88,60 @@ void shouldCollectExpiredJobs() {
     verify(jobExecutionService).deleteJobs(expiredJobs);
   }
 
+  @Test
+  void testResendJob() {
```

Comment:
I think if we will extract separate private methods for creating entitites - code will be more readable